classes/class3/mysimplehw/routes/cats.js

```diff
@@ -0,0 +1,103 @@
+var assert = require('assert');
+var catData = require('./catData');
+var mongojs = require('mongojs');
+var mongoURI = process.env.MONGOURI || "mongodb://localhost/test";
+
+// http://mafintosh.github.io/mongojs/
+var db = mongojs(mongoURI, ['cats']);
+
+// FIXME: should check for existence of collection rather than hard create
+db.createCollection('cats');
+
+var insert = function (req, res) {
+
+	// Create a random cat
+	var randCat = {
+		name: catData.names[Math.floor(Math.random()*catData.names.length)],
+		age: Math.ceil(Math.random()*100),
+		colors: (function() {
```

Comment:
Doing this as an IIFE is pretty unnecessary. Define the function outside the object and then assign it to the value of the key.